src/utils/logs.ts

```diff
@@ -926,6 +928,13 @@ export function logEntryCannotBeExternal(unresolvedId: string): RollupLog {
 	};
 }
 
+export function logExternalizedModulesCannotBeIncludedInManualChunks(source: string) {
+	return {
+		code: EXTERNALIZED_MODULES_CANNOT_BE_INCLUDED_IN_MANUAL_CHUNKS,
+		message: `"${source}" cannot be included in manualChunks, because it is resolved as an external module by plugins`
```

Comment:
Nice! Question: can this only happen when plugins make the module external, or could this also happen when a user uses the "external" option? Then maybe we should adjust the message:
 
 `"${source}" cannot be included in manualChunks, because it is resolved as an external module by the "external" option or plugins.`

Also note that error message usually end with a "." But this is just tiny details, nice work overall!

Reply:
> could this also happen when a user uses the "external" option?

Ok, I will give it a try.

Reply:
done